### DISCUSSION TITLE
Welcome to SymSpell Discussions!

### DISCUSSION THREAD

**Discussion by wolfgarbe** [wolfgarbe/symspell#103] [General]
## 👋 Welcome!
Discussions is the space to have conversations, ask questions and post answers without opening issues.
  * Ask questions you’re wondering about.
  * Share ideas.
  * Topics are SymSpell, spelling correction, auto completion and fuzzy search.

**Comment by desial** [comment]
Is there a version of this code in Python?

**Reply by wolfgarbe** [reply]
Yes, there are several Python ports: https://github.com/wolfgarbe/SymSpell#ports
